components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionBuilder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.management;
+
+import org.wso2.carbon.identity.action.management.dao.model.ActionDTO;
+import org.wso2.carbon.identity.action.management.model.Action;
+
+/**
+ * This interface defines the Action Resolver.
+ * Action Resolver is the component that is responsible for the conversions between Action and ExtendedAction
+ * objects.
+ */
```

Comment:
Can we say an Action Model converter

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6119/commits/73129ff409dc220674594045370c909805220d13